enhancements/installer/assisted-installer-in-cluster.md

```diff
@@ -0,0 +1,641 @@
+and installation tasks. It correlates to the "host" resource in the Assisted
+Installer's REST API.
+
+Spec:
+ * clusterName: string
+ * role: [auto-assign, master, worker]
+ * hostname: string
+ * machineConfigPool: string
+ * disks:
+   * diskId: string
+   * role: [none, install]
+ * enabled: boolean
+ * ignitionConfigOverrides: string
+ * installerArgs: string
+
+Status:
+ * state: string enum
+ * stateInfo: string
+ * stateUpdatedAt: string
+ * logsCollectedAt:string
+ * installerVersion: string
+ * createdAt: string
+ * updatedAt: string
+ * checkedInAt: string
+ * hostname: string
+ * bootstrap: boolean
+ * discoveryAgentVersion: string
+ * inventory:
+   * timestamp: string
+   * hostname: string
+   * bmcAddress: string
+   * bmcV6Address: string
+   * memory:
+     * physicalBytes: integer
+     * usableBytes: integer
+   * cpu:
+     * count: integer
+     * frequency: number
+     * flags: array of strings
+     * modelName: string
+     * architecture: string
+   * interfaces: array
+     * ipv6Addresses: array of strings
+     * vendor: string
+     * name: string
+     * hasCarrier: bool
+     * product: string
+     * mtu: integer
+     * ipv4Addresses: array of strings
+     * biosDevname: string
+     * clientId: string
+     * macAddress: string
+     * flags: array of strings
+     * speedMbps: integer
+   * disks: array
+     * driveType: string
+     * vendor: string
+     * name: string
+     * path: string
+     * hctl: string
+     * byPath: string
+     * model: string
+     * wwn: string
+     * serial: string
+     * sizeBytes: integer
+     * bootable: boolean
+     * smart: string
+   * boot:
+     * current_boot_mode:string
+     * pxeInterface:string
+   * system_vendor:
+     * serialNumber:string
+     * productName:string
+     * manufacturer: string
+     * virtual: boolean
+ * validation_info:
+   * hardware:
+     * hasInventory: [success, failure, pending, error]
+     * hasMinCpuCores: [success, failure, pending, error]
+     * hasMinMemory: [success, failure, pending, error]
+     * hasMinValidDisks: [success, failure, pending, error]
+     * hasCpuCoresForRole: [success, failure, pending, error]
+     * hasMemoryForRole: [success, failure, pending, error]
+     * isHostnameValid: [success, failure, pending, error]
+     * isHostnameUnique: [success, failure, pending, error]
+     * isPlatformValid: [success, failure, pending, error]
+   * network:
+     * isConnected: [success, failure, pending, error]
+     * isMachineCidrDefined: [success, failure, pending, error]
+     * belongsToMachineCidr: [success, failure, pending, error]
+     * isApiVipConnected: [success, failure, pending, error]
+     * belongsToMajorityGroup: [success, failure, pending, error]
+     * isNtpSynced: [success, failure, pending, error]
+  * progress:
+    * currentStage: enum
+    * progressInfo: string
+    * stageStartedAt: string
+    * stageUpdatedAt: string
+ * connectivity: array of
+   * host
```

Comment:
which object is this?

Reply:
probably this one https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L3541

Reply:
@filanov proposed to reference the Host object, which makes sense to me.
Please approve/disapprove